priv/repo/migrations/20240525065205_update_user_session.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Dbservice.Repo.Migrations.UpdateUserSession do
+  use Ecto.Migration
+
+  def change do
+    alter table(:user_session) do
+      add :user_activity_type, :string
```

Comment:
`subtype` also needs to be added here?

Reply:
User does not have a `subtype`. A session does.